packages/xo-web/src/xo-app/logs/index.js

```diff
@@ -342,161 +364,115 @@ const LOG_COLUMNS = [
           >
             {_('jobFinished')}
           </span>
+        ) : log.status === 'started' ? (
+          get(() => jobs[log.jobId].running) ? (
+            <span className='tag tag-warning'>{_('jobStarted')}</span>
+          ) : (
+            <span className='tag tag-danger'>{_('jobInterrupted')}</span>
+          )
+        ) : (
+          <span className='tag tag-default'>{_('jobUnknown')}</span>
```

Comment:
`jobUnknown` has no message.

Reply:
It was already here before, that's weird…